src/components/TodoCount/TodoCount.tsx

```diff
@@ -0,0 +1,13 @@
+import { Todo } from '../../types/Todo';
```

Comment:
I think no need to crate a separate component for this purpose, you could move it to `TodoFilter`.
What do you think about it?